src/main/java/com/linkedin/kmf/services/ClusterTopicManipulationService.java

```diff
@@ -45,22 +53,33 @@
   private final ScheduledExecutorService _executor;
   private final AdminClient _adminClient;
   private boolean _isOngoingTopicCreationDone;
+  private boolean _isOngoingTopicDeletionDone;
   private final AtomicBoolean _running;
   private String _currentlyOngoingTopic;
   int _expectedPartitionsCount;
   // TODO -- ClusterTopicManipulationMetrics implementation in progress!
-  private ClusterTopicManipulationMetrics _clusterTopicManipulationMetrics;
+  private final ClusterTopicManipulationMetrics _clusterTopicManipulationSensors;
 
   public ClusterTopicManipulationService(String name, AdminClient adminClient) {
     LOGGER.info("ClusterTopicManipulationService constructor initiated {}", this.getClass().getName());
 
     _isOngoingTopicCreationDone = true;
+    _isOngoingTopicDeletionDone = true;
     _adminClient = adminClient;
     _executor = Executors.newSingleThreadScheduledExecutor();
     _reportIntervalSecond = Duration.ofSeconds(1);
     _running = new AtomicBoolean(false);
     _configDefinedServiceName = name;
     // TODO: instantiate a new instance of ClusterTopicManipulationMetrics(..) here.
+
+    MetricConfig metricConfig = new MetricConfig().samples(60).timeWindow(1000, TimeUnit.MILLISECONDS);
```

Comment:
Should we make the 60 and 1000 here configurable? Or there is no need because it should always be using these numbers.

Reply:
Sounds good. Let me pull up a separate PR for that after this, since there are other things I want to make configurable also.